### ISSUE TITLE
problem with menues

### ISSUE THREAD

**Issue by jnz31** [PAYONE-GmbH/woocommerce-3#3]
activating BS PAYONE WooCommerce 3 in my installation wrecks the appearance > menus page. 

at the end of the html document it goes:
```
b>Notice</b>:  Trying to get property 'total' of non-object in <b>/.../wp-content/plugins/woocommerce/includes/abstracts/abstract-wc-payment-gateway.php</b> on line <b>263</b><br />
<br />
<b>Fatal error</b>:  Uncaught Error: Call to a member function get_billing_country() on null in /.../wp-content/plugins/payone-woocommerce-3/src/Payone/Gateway/GatewayBase.php:180
Stack trace:
#0 /.../wp-content/plugins/woocommerce/includes/class-wc-payment-gateways.php(159): Payone\Gateway\GatewayBase-&gt;is_available()
#1 /.../wp-content/plugins/woocommerce/includes/wc-account-functions.php(117): WC_Payment_Gateways-&gt;get_available_payment_gateways()
#2 /.../wp-content/plugins/woocommerce/includes/admin/class-wc-admin-menus.php(288): wc_get_account_menu_items()
#3 /.../wp-admin/includes/template.php(1183): WC_Admin_Menus-&gt;nav_menu_links(NULL, Array)
#4 /.../wp-admin/ in <b>/.../wp-content/plugins/payone-woocommerce-3/src/Payone/Gateway/GatewayBase.php</b> on line <b>180</b><br />
```

deactivating BS PAYONE WooCommerce 3 solves the issue.

**Comment by jnz31** [comment]
i kinda solved it by changing WC()->customer->get_billing_country() to WC()->customer in the elseif condition. not sure, if this works now properly, but at least the menu page is now back up..

`if ( $order_id ) {
  $order   = wc_get_order( $order_id );
  $country = (string) $order->get_billing_country();
} elseif ( WC()->customer ) {
  $country = (string) WC()->customer->get_billing_country();
}`

**Comment by fjbender** [comment]
We applied this fix in e039959d6fc5bfba4e2dc95d0b1c5f3f825b049c, will be included in Release v1.3.10

@hreinberger Please prepare release v1.3.10 accordingly to close this issue, thanks!

**Comment by jnz31** [comment]
hi all
installed your fix and the menu page works as expected.

thanks
